admin/app/pagepresser/NextGenInteractiveHtmlCleaner.scala

```diff
@@ -0,0 +1,24 @@
+package pagepresser
+
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+
+object NextGenInteractiveHtmlCleaner extends HtmlCleaner with implicits.WSRequests {
+
+  override def canClean(document: Document): Boolean = {
+    document.getElementById("interactive-content") != null &&
+      document.getElementsByAttributeValue("rel","canonical").attr("href").toLowerCase.contains("/ng-interactive/")
+  }
+
+  override def clean(document: Document, convertToHttps: Boolean) = {
+    universalClean(document)
+    //removeScripts(document)
```

Comment:
I thought it made it a little more obvious what the main difference is between this and the other interactive cleaner. Might also be able to revisit once we've got a few working examples. But could go for now I guess.